### PR TITLE
Transparent popup

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
### Description of Change ###

Set background view type for popup controller

 ### Linked Issues ###
 

 https://github.com/CommunityToolkit/Maui/issues/548
https://github.com/CommunityToolkit/Maui/issues/499
https://github.com/CommunityToolkit/Maui/issues/800

 ### PR Checklist ###
 
 - [x] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [x] Has samples (if omitted, state reason in description)
 - [x] Rebased on top of `main` at time of PR
 - [x] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

### CODE REVIEW

File: src/CommunityToolkit.Maui.Core/Views/Popup/PopupExtensions.macios.cs [hunk 1]
```diff
@@ -110,25 +111,47 @@ public static void SetLayout(this MauiPopup mauiPopup, in IPopup popup)
...
+			BackgroundColor = Colors.Transparent.ToPlatform();
+			Alpha = 0.0f;
+		}
+
+		public override NFloat ArrowOffset { get; set; }
+
+		public override UIPopoverArrowDirection ArrowDirection { get; set; }
+
+		[Export("arrowHeight")]
+		static new float GetArrowHeight()
```
Comment (pictos) [inline src/CommunityToolkit.Maui.Core/Views/Popup/PopupExtensions.macios.cs:140]: What are those arrow methods? Is the arrow that appears when the popup is anchored to a view?

Comment (VladislavAntonyuk) [reply src/CommunityToolkit.Maui.Core/Views/Popup/PopupExtensions.macios.cs:140]: This is a requirement of UIPopoverBackgroundView to add these methods.
They used to create absolutely custom background (e.g. add image).

**Review by brminnick — APPROVED** [review]

**Comment by samir05051994** [comment]
hi @jfversluis @brminnick 
May I know Please when this issue will be fixed?
This issue is also happening with https://www.nuget.org/packages/Mopups/

**Comment by samir05051994** [comment]
Hi CommunityToolkit/Maui

Thanks for the update.

Regards and Thanks
Samir

**Comment by borrmann** [comment]
how is this used? Just by setting Color="Transparent" in Xaml? 
That still doesnt work in my application.

**Pull request merged**